test/test_vending_machine.py

```diff
@@ -44,17 +49,19 @@ def test_buy_product_with_payment_expects_product(self):
         result = self.vending_machine.buy_product()
 
         # Assert
+        assert_is_not_none(result)
         assert_that(result, is_(not_none()))
 
-    @raises(RuntimeError)
+    #@raises(RuntimeError)
```

Comment:
Ok, that rationale makes sense to me.  Will update the presentation then merge the PR.